interp/notation_ops.ml

```diff
@@ -1344,7 +1362,7 @@ let rec match_ inner u alp metas sigma a1 a2 =
 
   (* Matching compositionally *)
   | GVar id1, NVar id2 when alpha_var id1 id2 (fst (snd alp)) -> sigma
-  | GRef (r1,_), NRef r2 when (GlobRef.equal r1 r2) -> sigma
+  | GRef (r1,u1), NRef (r2,u2) when (GlobRef.equal r1 r2) && compare_glob_universe_instances true (ref false) u1 u2 -> sigma
```

Comment:
As a last comment, if you don't mind, maybe give a name to this function, earlier, say `glob_universe_instances_le` or `glob_universe_instances_included`, or `glob_universe_instances_less_or_equal` or whatever, as a way to comment what the combination of `true` and `ref false` is doing. This abbreviation could be just after the definition of `glob_universe_instances`.

Reply:
That is a good idea. Done.